source/AsyncRgbLedSimulationDataGenerator.cpp

```diff
@@ -1,11 +1,22 @@
 #include "AsyncRgbLedSimulationDataGenerator.h"
+
+#include <cmath> // for M_PI, cos
+#include <iostream>
+
 #include "AsyncRgbLedAnalyzerSettings.h"
 
 #include <AnalyzerHelpers.h>
 
+// these values are from the WS2811, in high-speed mode
```

Comment:
This is fine for now, but later we'll want to replace this with a table of data that contains the details for all controllers. We'll also want to include in comments the URLs of the sources of this information.